### PR TITLE
CMB-586: add booklets CRUD documentation

### PR DESCRIPTION
**Pull request by sachinmurali**
Fixes [CMB-586](https://lobsters.atlassian.net/browse/CMB-586)

## Checklist

- [ ] Up to date with `main`
- [ ] All the tests are passing
  - [ ] Delete all resources created in tests
- [ ] Prettier
- [ ] Spectral Lint
- [ ] `npm run bundle` outputs nothing suspect
- [ ] `npm run postman` outputs nothing suspect

## Changes

- Adds booklets CRUD documentation
  - The fields to create a booklet and that are present in the response body are in accordance with the fields listed in this google doc - https://docs.google.com/document/d/1bogXv4M2eAwvNFx6QvkakBVAqN35x4Rk8rKP4C9Uetc/edit
  - The booklet fields are a subset of the letter fields
  - Booklets are always printed in color. Hence the default is `true`.
  - The `size` property defaults to `9x6`.
- The code samples are not added because the booklets resource does not exist in the SDKs. Once the SDK is updated with the booklet resource, the code samples will need to be introduced.

## Areas of Concern (optional)

As mentioned above, the code samples are missing. This is because they are not part of the SDKs. Once they are introduced in the SDKs, the docs will need to be updated. 

 I encourage the reviewers to pull down the branch and preview the HTML to catch any errors or mistakes prior to approval.

[CMB-586]: https://lobsters.atlassian.net/browse/CMB-586?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by sachinmurali** [comment]
The booklets launch is scheduled for end of July. This PR is expected to be merged when we're good to go with the launch of booklets. However, at this moment in time, it is good for a review.

**Review by mrjamesjcho — COMMENTED** [review]
A couple of notes but looking good so far!

**Review by mrjamesjcho — APPROVED** [review]
Just need to remove the `color` property in the response but otherwise looks good 😄. I'll approve to not block.

**Review by bethqiang — APPROVED** [review]

File: resources/booklets/booklets.yml [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+          -d 'merge_variables[name]=Harry' \
+          -d 'qr_code[position]=relative' \
+          -d 'qr_code[redirect_url]=https://www.lob.com' \
+          -d 'qr_code[width]=2' \
+          -d 'qr_code[bottom]=2' \
+          -d 'qr_code[left]=2' \
+          -d 'qr_code[pages]=3,4-5' \
+          -d 'fsc=true'
+      label: CURL
+  #   - lang: Typescript
```
[the remark this answers is not in the document]
Comment (sachinmurali) [reply resources/booklets/booklets.yml:213]: Booklets are not part of the SDKs. The SDKs are in a strange state where they have not been maintained over the past several months. I've surfaced this to my manager @nperri6 (and @gtstewart16) and he's been in talks with the ELT to determine the next steps for the SDKs. In the meantime, we do not want to convey the wrong information to the users of the API, given that the examples are pointing to the usage of these objects under the respective SDKs. That's why the Blueprint team agreed to leave out the code examples.

[the remark this answers is not in the document]
Comment (bethqiang) [reply resources/booklets/booklets.yml:213]: got it, makes sense!

**Pull request merged**